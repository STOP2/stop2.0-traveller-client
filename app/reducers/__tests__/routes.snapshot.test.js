import routesReducer, { initialState } from '../../reducers/routes';

describe('routes reducer', () => {
  it('returns the same state on an unhandled action', () => {
    expect(routesReducer(initialState, { type: '_NULL' })).toMatchSnapshot();
  });
});
