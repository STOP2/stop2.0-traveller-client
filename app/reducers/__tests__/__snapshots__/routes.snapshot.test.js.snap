exports[`routes reducer returns the same state on an unhandled action 1`] = `
Object {
  "scene": Object {},
}
`;
